src/main/java/ee/tuleva/onboarding/fund/FundController.java

```diff
@@ -21,8 +22,12 @@
 
     @ApiOperation(value = "Get info about available funds")
     @RequestMapping(method = GET, value = "/funds")
-    public List<FundResponse> get(@RequestParam("fundManager.name") Optional<String> fundManagerName) {
-        return fundService.getFunds(fundManagerName);
+    public List<FundResponse> get(@RequestParam("fundManager.name") Optional<String> fundManagerName,
+                                  @RequestHeader(value = "Accept-Language") String language
+    ) {
+
+        List<FundResponse> funds = fundService.getFunds(fundManagerName, language == null ? "et" : language.toLowerCase());
```

Comment:
Extract the `language == null ? "et" : language.toLowerCase()` part to a variable